app/views/unattended/provisioning_templates/snippet/fist_boot_setup_script.erb

```diff
@@ -0,0 +1,9 @@
+<%#
+kind: snippet
+name: first_boot_setup_script
+model: ProvisioningTemplate
+snippet: true
+description: |
+    First boot setup script
+-%>
+<%= indent(2, skip1: true) { snippet('built', :variables => { :endpoint => 'built', :method => 'POST', :body_file => '/root/install.post.log' }) } -%>
```

Comment:
The service will always trigger when rebooting ? So maybe systemctl disable ... ?

Reply:
Yes, you are right. Thanks for the suggestion.